### PR TITLE
Adding content for what is a CNAB

### PR DESCRIPTION
**Pull request by jeremyrickard**
Adding some content for the what is a CNAB portion of the workshop

### CODE REVIEW

**Comment by jeremyrickard** [comment]
Preview at: https://deploy-preview-393--porter.netlify.com/pack-your-bags/#1

File: docs/content/slides/pack-your-bags/index.md [hunk 1]
```diff
@@ -61,49 +61,359 @@ name: introductions
...
+# Your App...
 
 ---
-# Breakdown of Azure MySQL Wordpress
+
+# The Invocation Image
+
+--
+
+* MSI for the Cloud?
```
Comment (carolynvs) [inline docs/content/slides/pack-your-bags/index.md]: Can you tell why this isn't rendering as a normal bullet point? It's showing up super funky with an astrix and in a larger font.

File: docs/content/slides/pack-your-bags/index.md [hunk 2]
```diff
@@ -61,49 +61,356 @@ name: introductions
...
+# Suddenly McGuffin has FANS!
+
+.center[
+  ![all the github stars!!!](/images/pack-your-bags/your-fans.jpg)
+]
+
+---
+class: center, middle
+
+# Everyone do this is not going to cut it...
```
Comment (carolynvs) [inline docs/content/slides/pack-your-bags/index.md]: Did you mean "This is too damn hard"? or "This won't scale" or ? What's there doesn't read well

File: docs/content/slides/pack-your-bags/index.md [hunk 3]
```diff
@@ -61,49 +61,356 @@ name: introductions
...
+  ![nobody wants to do this](/images/pack-your-bags/scroll-of-truth.png)
+]
+
+---
+class: center, middle
+
+# So what do we do...
+
+.center[
+  ![this is my thinking face](/images/pack-your-bags/thinking.jpg)
```
Comment (carolynvs) [inline docs/content/slides/pack-your-bags/index.md]: suggestion: make this a bit smaller, kids head is huge compared to the other pics

File: docs/content/slides/pack-your-bags/index.md [hunk 4]
```diff
@@ -61,49 +61,356 @@ name: introductions
...
+]
+
+---
+
```
Comment (carolynvs) [inline docs/content/slides/pack-your-bags/index.md]: suggestion: center this like you did the other slides with 

```
class: center
```

**Comment by jeremyrickard** [comment]
@carolynvs thy will be done!

**Pull request merged**